addons/mail/static/src/owl/components/chatter_topbar/chatter_topbar.js

```diff
@@ -28,15 +29,68 @@ class ChatterTopbar extends Component {
 
     /**
      * @private
-     * @param {Event} ev
+     * @param {MouseEvent} ev
      */
     _onClickAttachments(ev) {
         this.trigger('o-chatter-topbar-select-attachment');
     }
+
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onClickFollow(ev) {
+        this.trigger('o-chatter-topbar-follow');
+    }
+
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onClickFollowers(ev) {
+        this.trigger('o-chatter-topbar-show-followers');
+    }
+
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onClickLogNote(ev) {
+        this.trigger('o-chatter-topbar-log-note');
+    }
+
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onClickScheduleActivity(ev) {
+        this.trigger('o-chatter-topbar-schedule-activity');
+    }
+
+    /**
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _onClickSendMessage(ev) {
+        this.trigger('o-chatter-topbar-send-message');
+    }
 }
 
+ChatterTopbar.defaultProps = {
+    isDisabled: false,
+};
+
 ChatterTopbar.props = {
-    threadLocalId: String,
+    isComposerLog: Boolean,
+    isComposerVisible: Boolean,
+    isDisabled: {
+        type: Boolean,
+        optional: true,
```

Comment:
should not be necessary, I think defaults are applied before validation

Reply:
you mean the "optionability" is not necessary ?
so 
```
isDisabled: Boolean
```
?

Reply:
yup, it computes defaultProps before checking props, and since defaultProps is a boolean, should not have to tag it as optional

Reply:
Ok didn't know that, thanks !